scripts/check_coverage.sh

```diff
@@ -17,17 +16,13 @@ trap finish EXIT ERR INT TERM
 #start script logic
 OUTPUT=`./scripts/coverage.sh`
 
-while read -r LINE; do
+while IFS= read -r LINE; do
   echo -e "$LINE"
   if [[ $LINE =~ "%" ]]; then
     PERCENT=$(echo "$LINE"|cut -d: -f2-|cut -d% -f1|cut -d. -f1|tr -d ' ')
-    if [[ $PERCENT -lt 30 ]]; then
-      CHECKCOV=true
+    if [[ $PERCENT -lt $COV_MIN ]]; then
+      echo "Package has less than $COV_MIN% code coverage. Run ./scripts/coverage.sh --html to see a detailed coverage report, and add tests to improve your coverage"
```

Comment:
If you move this message+exit inside the loop, it'll only report on the first package that falls below the coverage threshold.  It won't initially indicate that other packages are below, you'd only see those once you fix the coverage for this first package.

Is that acceptable?  

Also, I think that the `$COV_MIN%` would render the text as you want (eg 30%), but normally the variables are encased like this `${COV_MIN}%` to mesh with the rest of the string.

Reply:
yeah. I think this is at least better than what it was. Probably the ideal state is to store all the packages which need more coverage, and summarize them at the end, but... I think that's a little out of scope for this PR.

I'll fix `${COV_MIN}%`... that's a good point.